libraries/AP_Scripting/examples/Mission_test.lua

```diff
@@ -0,0 +1,49 @@
+-- This script is a test for AP_Mission bindings
+
+local last_mission_index = mission:get_current_nav_index()
+
+function update() -- this is the loop which periodically runs
+
+  local mission_state = mission:state()
+
+  -- make sure the mission is running
+  if mission_state == 2 then --MISSION_COMPLETE
```

Comment:
This is where the enums are not working as expected, if I swap the 2 for MISSION_COMPLETE it runs but the if doesn't work.